app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (f"{{Name:"
+                f" {self.name},"
+                f" Health: {self.health},"
+                f" Hidden: {self.hidden}}}"
+                )
```

Comment:
In case you do not place any code on the same line with a closing parenthesis, you should not place the part of the string on the same line with the opening parenthesis. Besides that, correct your indentation so it would be seen that the string is inside of parentheses and not on the same indentation level.